modmesh/app/euler1d.py

```diff
@@ -132,13 +132,40 @@ class SolverConfig():
           table.
         - :meth:`columnCount()`: Get the number of columns in the
           configuration table.
-        - :meth:`get_var(key_row, key_col)`: Get the value of a configuration
-          variable based on its key.
     """
-    def __init__(self, input_data):
+    class Accessor:
+        """
+        Helper calss to access data within the configuration table using
+        multiple dimensions, currenlty only support 2-dimensions table.
+
+        This class allows users to access data by chaining multiple [],
+        supporting string-based.
+        """
+        def __init__(self, data, dimIdx=0, header=None):
+            self._data = data
+            self._header = header
+            self._dimIdx = dimIdx
+
+        def __getitem__(self, key):
+            if self._dimIdx == 0:
```

Comment:
This is used to identify whether the `[]` operator is currently handling row accessing, because row access requires checking the row name in the first column.